satpy/readers/generic_image.py

```diff
@@ -31,6 +31,7 @@
 import dask.array as da
 import numpy as np
 import rasterio
+import rioxarray  # noqa: F401, need by xarray with the engine rasterio
```

Comment:
This comment uses a `,` to separate the rule from the description, but the other one uses a `:`. What is the recommended way of doing this (I assume it is supported)?

Reply:
Right, the `,` can be used when multiple rules should be ignored. i.e. : `x=5  # noqa: E731,E123`  So If I want to be fully correct I should stick with some dot `.` or simply a space.

Moreover, as [stated in the documentation](https://flake8.pycqa.org/en/latest/user/violations.html#in-line-ignoring-errors) : 

> Contents before and after the `# noqa: ...` portion are ignored so multiple comments may appear on one line. Here are several examples:
> ```python
># mypy requires `# type: ignore` to appear first
>x = 5  # type: ignore  # noqa: ABC123
># can use to add useful user information to a noqa comment
>y = 6  # noqa: ABC456  # TODO: will fix this later
>```